requirements.txt

```diff
@@ -6,4 +6,5 @@ jax==0.2.19
 jaxlib==0.1.70
 flax
 bm3d
-svmbir
+# svmbir
+git+https://github.com/cabouman/svmbir.git@nanfix
```

Comment:
When this is merged, please add an issue as a reminder to revert this to just `svmbir` as soon as the fix is released.